Resources/public/Admin.js

```diff
@@ -69,6 +71,45 @@ var Admin = {
             overflow: 'scroll'
         });
     },
+    
+    setup_select: function(subject) {
+       jQuery('select', subject).each(function() {
+            var select = jQuery(this);
+            var fieldContainer = select.closest('div[class="field-container"]');
+            var editBtn = fieldContainer.find('a[class~="sonata_model_edit_btn"]');
```

Comment:
Please use proper class selector, do not overspecify, and respect the current case convention (kebab-case).
```js
var editBtn = fieldContainer.find('.sonata-type-model-action-edit');
```